client/blocks/signup-form/crowdsignal.jsx

```diff
@@ -0,0 +1,171 @@
+class CrowdsignalSignupForm extends Component {
+
+	static propTypes = {
+		// formFields: PropTypes.,
+		disabled: PropTypes.bool,
+		handleSocialResponse: PropTypes.func,
+		handleSubmit: PropTypes.func,
+		isSocialSignupEnabled: PropTypes.bool,
+		loginLink: PropTypes.string,
+		oauth2Client: PropTypes.object,
+		// socialService: PropTypes.,
+		// socialServiceResponse: PropTypes.
+		recordBackLinkClick: PropTypes.func,
+		submitting: PropTypes.bool,
+		translate: PropTypes.func.isRequired,
+	};
+
+	defaultProps = {
+		disabled: false,
+		submitting: false,
+	};
+
+	state = {
+		showSignupForm: false,
+	};
+
+	showSignupForm = () => this.setState( { showSignupForm: true } );
+
+	hideSignupForm = () => this.setState( { showSignupForm: false } );
+
+	renderLearnMoreLink() {
+		const { translate } = this.props;
+		const link = (
+			<a
+				href="https://crowdsignal.com/2012/11/12/polldaddy-using-wordpress-com-logins/"
+				target="_blank"
+				rel="noopener noreferrer"
+			/>
+		);
+
+		return (
+			<p className="signup-form__crowdsignal-learn-more">
+				{ translate( 'Why WordPress.com? {{a}}Learn more{{/a}}.', {
+					components: {
+						a: link
+					}
+				} ) }
+			</p>
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		const socialCardClass = classNames( 'signup-form__crowdsignal-card', 'signup-form__crowdsignal-social', {
+			'is-active': ! this.state.showSignupForm,
+		} );
+		const signupCardClass = classNames( 'signup-form__crowdsignal-card', {
+			'is-active': this.state.showSignupForm,
+		} );
+		const backButtonWrapperClass = classNames( 'signup-form__crowdsignal-back-button-wrapper', {
+			'is-first-step': ! this.state.showSignupForm,
+		} );
+
+		return (
+			<div className="signup-form__crowdsignal">
+				<div className="signup-form__crowdsignal-layout">
+					<div className={ socialCardClass }>
+						<h2 className="signup-form__crowdsignal-card-header">{ translate( 'Connect an existing account:' ) }</h2>
+
+						<div className="signup-form__crowdsignal-card-content">
+							<p className="signup-form__crowdsignal-card-subheader">
+								{ translate( 'The fastest way.{{br/}}Use one of your existing accounts{{br/}}to sign up with Crowdsignal:', {
+									components: {
+										br: <br />
+									}
+								} ) }
+							</p>
+
+							<Button primary href={ this.props.loginLink } className="signup-form__crowdsignal-wpcom">
+								<Gridicon icon="my-sites" size={ 20 } />
+								<span>{ translate( 'Sign up with WordPress.com' ) }</span>
+							</Button>
+							{ this.props.isSocialSignupEnabled && (
+								<SocialSignupForm
+									compact
+									handleResponse={ this.props.handleSocialResponse }
+									socialService={ this.props.socialService }
+									socialServiceResponse={ this.props.socialServiceResponse }
+								/>
+							) }
+							<Button className="signup-form__crowdsignal-show-form" onClick={ this.showSignupForm }>
+								{ translate( 'Create a WordPress.com Account' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Create WordPress.com account' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/crowdsignal-signup-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).